commands/show.go

```diff
@@ -68,14 +140,76 @@ Timestamp:  %s
 				strconv.FormatUint(uint64(block.Nonce), 10),
 				strconv.FormatUint(uint64(block.Timestamp), 10),
 			)
-			if err != nil {
-				return err
+			return err
+		}),
+	},
+}
+
+var showMessagesCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Show a filecoin message collection by its CID",
```

Comment:
I think it would be helpful to add a quick line about how to get the message collection CID, similarly do the same with showReceiptsCmd.  Just something like "(obtained from the Block Header)" or something like that.